backend/src/main/java/reviewme/review/service/ReviewService.java

```diff
@@ -93,4 +95,30 @@ private ReviewSetupResponse createReviewSetupResponse(ReviewGroup reviewGroup) {
         return new ReviewSetupResponse(reviewGroup.getReviewee(), reviewGroup.getProjectName(),
                 questionSetupRespons, reviewSetUpKeywordRespons);
     }
+
+    @Transactional(readOnly = true)
+    public ReceivedReviewsResponse findReceivedReviews(String groupAccessCode) {
+        ReviewGroup reviewGroup = reviewGroupRepository.findByGroupAccessCode(groupAccessCode)
+                .orElseThrow(InvalidGroupAccessCodeException::new);
+        List<ReceivedReviewResponse> reviewResponses = reviewRepository.findAllByReviewGroupId(reviewGroup.getId())
+                .stream()
+                .map(this::extractResponse)
+                .toList();
+        return new ReceivedReviewsResponse(reviewGroup.getReviewee(), reviewGroup.getProjectName(), reviewResponses);
+    }
```

Comment:
(단순 제안) 변수 선언 사이, 리턴문 전 부분들에 개행 하나 해주면 어떨까요?
```suggestion
    @Transactional(readOnly = true)
    public ReceivedReviewsResponse findReceivedReviews(String groupAccessCode) {
        ReviewGroup reviewGroup = reviewGroupRepository.findByGroupAccessCode(groupAccessCode)
                .orElseThrow(InvalidGroupAccessCodeException::new);
        List<ReceivedReviewResponse> reviewResponses = 
        reviewRepository.findAllByReviewGroupId(reviewGroup.getId())
                .stream()
                .map(this::extractResponse)
                .toList();
                
        return new ReceivedReviewsResponse(reviewGroup.getReviewee(), reviewGroup.getProjectName(), reviewResponses);
    }
```